config/initializers/app_routes.rb

```diff
@@ -2,6 +2,6 @@ module AppRoutes
   # memoize this due to a bug in MRI that shows up in multi-threaded code
   # see https://github.com/puma/puma/issues/647, search for "bug in MRI"
```

Comment:
the linked bug is: https://bugs.ruby-lang.org/issues/10871 seems closed in ruby 2.2